test/es6/rlexe.xml

```diff
@@ -689,6 +689,18 @@
   </test>
   <test>
     <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -args summary -endargs</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -nodeferparse -args summary -endargs</compile-flags>
```

Comment:
Can we use the phase-style switches -off:deferparse and -force:deferparse instead of the flag-style -nodeferparse and -forcedeferparse, which I'd really like to retire at some point?